protocol/app/ante/market_update.go

```diff
@@ -0,0 +1,149 @@
+package ante
```

Comment:
nit: Leave a comment indicating this is a stop-gap measure before generalized functionality is added to the marketmap module to allow delaying / gating certain updates?